docs/en/ros2/px4_ros2_control_interface.md

```diff
@@ -350,13 +350,15 @@ The other setpoint types are currently experimental, and can be found in: [px4_r
 You can add your own setpoint types by adding a class that inherits from `px4_ros2::SetpointBase`, sets the configuration flags according to what the setpoint requires, and then publishes any topic containing a setpoint.
 :::
 
-#### Go-to Setpoint (GotoSetpointType)
+#### Go-to Setpoint (MulticopterGotoSetpointType)
```

Comment:
FYI @bkueng I figure it is worth updating the heading as well.

Note, there is also `MulticopterGotoGlobalSetpointType` which perhaps we should also mention?

Reply:
Good point, I added it